docs/README.md

```diff
@@ -87,7 +82,26 @@ Overview
     * overview of secret storage, SSH credentials etc.
 
 
-### Workflows ###
+## Inspection
 
-* [E2E testing with `kops` clusters](testing.md)
-* [Getting started on AWS](aws.md)
+* [Download `kops` configuration](download_config.md)
+    * methods to download the current generated `kops` configuration
+* [Get AWS subdomain NS records](ns.md)
+
+
+## Development
+
+* [E2E testing with `kops` clusters](development/testing.md)
+* [Developing using Docker](development/Docker.md)
+* [Updating Go Dependencies](development/dependencies.md)
+* [Hack Directory](development/hack.md)
```

Comment:
Alphabetic sorting would make this list easier to read and quicker to skim through, same goes for the other lists in the index (except perhaps "Quick start").